addons/account/models/account_tax.py

```diff
@@ -759,3 +1144,14 @@ def _compute_tax_id(self):
     def _onchange_repartition_type(self):
         if self.repartition_type == 'base':
             self.account_id = None
+
+    def _get_business_tax_account(self):
+        """ Get the default tax account to set on a business line.
```

Comment:
Business line ? I don't think introducing this new terminology helps :/ Plus, for me : business line => business object => invoices, SO, PO. And this account is actually the one to use behind business objects, on move lines. 

_get_aml_target_tax_account would do the job, I guess.